### ISSUE TITLE
allow certain accounts to load others' machines

### ISSUE THREAD

**Issue by CSchank** [CSchank/finsm#93]
For example, a person with a "TA" account could load a student's machine to help them with it.

**Comment by Necried** [comment]
This seems easily extendable with Django as our back-end currently.

**Comment by CSchank** [comment]
@Necried Indeed, I think we could add a special type of filter for accounts with access that would list everyone's machines. If we did that we might want a search functionality. Or we could provide a link-sharing system that only people with permission can open.
